### PR TITLE
Add feature for viewing inactive HRs

### PR DESCRIPTION
**Pull request by bmerchant22**

### CODE REVIEW

File: company/db.hr.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package company
 import "github.com/gin-gonic/gin"
 
 func getAllHRUserDB(ctx *gin.Context, HRs *[]CompanyHR) error {
-	tx := db.WithContext(ctx).Select("ID","CreatedAt","UpdatedAt","DeletedAt","Name","Email","Phone").Find(HRs)
+	tx := db.WithContext(ctx).Select("ID", "CreatedAt", "UpdatedAt", "DeletedAt", "Name", "Email", "Phone").Find(HRs)
```
Comment (1-Harshit) [inline company/db.hr.go:6]: Are we selecting everything here?

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (1-Harshit) [inline .idea/modules.xml:1]: Don't

File: company/model.go [hunk 3]
```diff
@@ -12,6 +12,7 @@ type Company struct {
 
 type CompanyHR struct {
 	gorm.Model
+	DeletedAt   gorm.DeletedAt
```
Comment (1-Harshit) [inline company/model.go:15]: 😐

File: container/Dockerfile [hunk 4]
```diff
@@ -17,7 +17,7 @@ RUN go mod download && go mod verify
 
 COPY . .
 
-RUN cp $GOPATH/src/github.com/spo-iitk/ras-backend/secret.yml.template $GOPATH/src/github.com/spo-iitk/ras-backend/secret.yml
```
Comment (1-Harshit) [inline container/Dockerfile:20]: Why?

File: docker-compose.yml [hunk 5]
```diff
@@ -22,7 +22,7 @@ services:
     networks:
       - default
     ports:
-      - "80"
+      - "80:80"
```
Comment (1-Harshit) [inline docker-compose.yml:25]: Why(2)?

**Review by 1-Harshit — COMMENTED** [review]
Why do we need this, again?

**Comment by bmerchant22** [comment]
Sorry for all of that, we finally came to conclusion that this feature is not needed right now, so I am closing the PR

**Pull request closed without merging**